### ISSUE TITLE
Request to bot failed

### ISSUE THREAD

**Issue by Herobr1ne** [marceltransier/ttt_discord_bot#16]
### Description

[TTT Discord Bot] Request to bot failed. Is the bot running?
Err: invalid url

### Steps to Reproduce

Nach Anleitung installiert

**Expected behavior:** Vorschlag zu fixen

**Actual behavior:** Sobald man den !discord command eingibt

**Reproduces how often:** 100%

**OS:** Debian 10

**Config:**

```
{
  "discord": {
    "token": "***********************",
    "guild": "************************",
    "channel": "***********************"
  },
  "server": {
    "port": "37405"
  }
}

```

**Comment by Herobr1ne** [comment]
Alles Lokal auf einem Server installiert

**Comment by Herobr1ne** [comment]
Ok hab es hinbekommen. Man muss bei den gmod startparametern -allowlocalhttp eingeben. Füg das bitte mal in deine Anleitung hinzu

**Comment by smallerYeet** [comment]
Hey, könntest du mir helfen den Bot zum Laufen zu bringen?
Habe leider keine Erfahrung mit sowas und bräuchte Hilfe
(Mein Discord ist smaller | Darius#7889)

**Comment by Herobr1ne** [comment]
@smallerYeet hast du dir das schon angeschaut?
http://steamcommunity.com/sharedfiles/filedetails/?id=1343501656

**Comment by smallerYeet** [comment]
Natürlich, nur weiß ich nicht wie man es anfängt. Also bei den ersten Sachen hackts schon

**Comment by Herobr1ne** [comment]
Kann dir gerne Helfen aber dafür müsstest du schon meine Freundschaftsanfrage annehmen xD Hab die vor 8 Tagen geschickt aber du hattest nie angenommen
